addons/l10n_uy/tests/test_doc_types.py

```diff
@@ -0,0 +1,90 @@
+from odoo import Command
+from odoo.tests.common import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("-at_install", "post_install", "post_install_l10n")
+class TestDocTypes(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='uy'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.partner_test = cls.env["res.partner"].create({
+            "name": "IEB Internacional",
+            "l10n_latam_identification_type_id": cls.env.ref("l10n_uy.it_rut").id,
+            "vat": "218435730016",
+            "street": "Bach 0",
+            "city": "Aeroparque",
+            "state_id": cls.env.ref("base.state_uy_02").id,
+            "country_id": cls.env.ref("base.uy").id,
+            "email": "rut@example.com",
+        })
+
+        cls.service_vat_22 = cls.env["product.product"].create({
+            "name": "Virtual Home Staging (VAT 22)",
+            "list_price": 38.25,
+            "standard_price": 45.5,
+            "type": "service",
+            "default_code": "VAT 22",
+        })
+
+        cls.invoice = cls._create_invoice(
+            l10n_latam_document_type_id=cls.env.ref("l10n_uy.dc_e_inv_exp").id,
+        )
+        cls.invoice.action_post()
+
+    @classmethod
+    def _create_invoice(cls, **kwargs):
+        kwargs = {**kwargs} if kwargs else {}
+        default_values = {
+            "partner_id": cls.partner_test.id,
+            "move_type": "out_invoice",
+        }
+        default_values.update(kwargs)
+        invoice = cls.env["account.move"].with_context(
+            default_move_type=default_values['move_type']).create(default_values)
+
+        invoice.write({"invoice_line_ids": [
+            Command.create(line_values)
+            for line_values in [{
+                "product_id": cls.service_vat_22.id,
+                "quantity": 1.0,
+                "price_unit": 100.0,
+                "display_type": "product",
+            }]]})
+
+        return invoice
+
+    def _create_credit_note(self, invoice):
+        refund_wizard = self.env["account.move.reversal"].with_context({
+            "active_ids": [invoice.id], "active_model": "account.move"}).create({
+                "reason": "Mercadería defectuosa",
+                "journal_id": invoice.journal_id.id
+            })
+        res = refund_wizard.refund_moves()
+
+        refund = self.env["account.move"].browse(res["res_id"])
+        return refund
+
+    def _create_debit_note(self, invoice):
+        debit_note_wizard = self.env["account.debit.note"].with_context(
```

Comment:
indent is a bit weird here, can you do the same as for the _create_credit_note function ? 

Also do you really need to go through the wizard to create the moves ? Is it not possible to use the init_invoice function with a move type that you need ? (apply to the 3 functions)